example/examples/CoverFlowExample.js

```diff
@@ -19,14 +28,15 @@ var CoverFlowExample = React.createClass({
         require('image!8-vine1'),
       ],
     };
-  },
+  }
+
   render() {
     return (
       <CoverFlow>
         {this.state.images.map((src, i) => <Image key={i} source={src} />)}
       </CoverFlow>
     );
   }
-});
+}
 
-module.exports = CoverFlowExample;
+export default CoverFlowExample;
```

Comment:
i'm cool with converting everything to ES6, but let's stick with `module.exports = Foo`. It's more compatible